CS_UCDs.ipynb

```diff
@@ -152,14 +149,16 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "coord = SkyCoord.from_name(\"m83\")\n",
-    "query=f\"select top 10 {ra_name}, {dec_name} from dbo.DetailedCatalog\"\n",
     "#  Look for all TAP services with x-ray and optical data\n",
     "collection={}\n",
     "for s in vo.regsearch(servicetype='tap',keywords=['x-ray','optical']):\n",
-    "    if \"wfau\" in s.ivoid:  continue #  These sometimes have issues\n",
+    "    #if \"wfau\" in s.ivoid:  continue #  These sometimes have issues\n",
```

Comment:
The `try-except` below makes this line unnecessary so it might be clearer to delete instead of commenting out.